packages/primitives/radio/meta.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { TComponentConfig } from 'autoprops'
+import { component, startWithType } from 'refun' // mapState,
+import { RadioInput, RadioGroup, TRadioInput, TRadioGroup } from './src'
+
+export const Component = component(
+  startWithType<TRadioInput & TRadioGroup>()
+)(({ initialValue, ...radioInputProps }) => {
+  return (
+    <RadioGroup
+      initialValue={initialValue}
+    >
+      <RadioInput {...radioInputProps}/>
+      <RadioInput {...radioInputProps} id="foo" value="test-foo"/>
+    </RadioGroup>
+  )
+})
+
+Component.displayName = 'RadioInput'
+
+export const config: TComponentConfig<TRadioInput & TRadioGroup> = {
+  props: {
+    initialValue: ['value'],
+    groupName: ['test-group'],
+    id: ['radio-test'],
+    isDisabled: [true],
+    value: ['value'],
```

Comment:
if a value is chosen, should it override the initialValue?
I'm asking because I ran it with 'test-foo' as the value but the initialValue of 'value' was still selected

Reply:
It was not my intention.
`initialValue` is merely the default value of the selection, it can even be empty

Reply:
in bubble-ui if you pass a value then it overrides the default one.
that's the only thing I'm not sure about in this pr, but we can wait for more reviews or get back to it later